docs/topics/downloader-middleware.rst

```diff
@@ -429,6 +429,40 @@ In order to use this policy, set:
 * :setting:`HTTPCACHE_POLICY` to ``scrapy.extensions.httpcache.RFC2616Policy``
 
 
+.. _httpcache-policy-custom:
+
+Writing your own policy
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can implement a cache policy by creating a class that
+defines the methods described below.
+
+.. module:: scrapy.extensions.httpcache
+
+.. class:: CachePolicy
+
+    .. method:: should_cache_request(request)
+
+      Return True if the :class:`~scrapy.http.Request` is cacheable otherwise False
```

Comment:
A few :lipstick: aesthetic suggestions for this and the following lines:

```suggestion
      Return ``True`` if the :class:`~scrapy.http.Request` is cacheable or ``False`` otherwise.
```